spec/tapioca/compilers/sorbet_spec.rb

```diff
@@ -0,0 +1,30 @@
+# typed: strict
+# frozen_string_literal: true
+
+require "spec_helper"
+
+class Tapioca::Compilers::SorbetSpec < Minitest::Spec
+  before do
+    @temp_env_value = T.let(nil, T.nilable(String))
+    @temp_env_value = ENV["TPC_SORBET_EXE"]
+  end
+
+  after do
+    ENV["TPC_SORBET_EXE"] = @temp_env_value
+  end
+
+  it("returns the value of TPC_SORBET_EXE if set") do
+    custom_path = 'bin/custom-sorbet-static'
+    ENV["TPC_SORBET_EXE"] = custom_path
+    sorbet_path = Tapioca::Compilers::Sorbet.sorbet_path
+    assert_equal(sorbet_path, custom_path)
+  end
+
+  it("returns the default sorbet path if TPC_SORBET_EXE is not set") do
+    ENV["TPC_SORBET_EXE"] = ''
+    gem_path = Gem::Specification.find_by_name("sorbet-static").full_gem_path
+    default_path = File.join(gem_path, "libexec", "sorbet")
```

Comment:
You should be able to get this more easily by:
```suggestion
    default_path = Tapioca::Compilers::Sorbet::SORBET
```

Reply:
Much easier! Slight modification, I made this `Tapioca::Compilers::Sorbet::SORBET.to_s.shellescape`, since `SORBET` is a `Pathname` and not a string.